Geometry/TrackerCommonData/data/PhaseII/TiltedTracker4025/pixfwd.xml

```diff
@@ -47,34 +47,10 @@ note: see OT365_200_IT4025_2016_12_06.cfg for full config files
 
 </ConstantsSection>
 <SolidSection label="pixfwd.xml">
- <!--<Polycone name="Forward" startPhi="0*deg" deltaPhi="360*deg" >
-   <ZSection z="[RootStartZ]" rMin="[cms:TrackBeamR1]" rMax="18.0*cm"/>
-   <ZSection z="0.02*cm" rMin="[cms:TrackBeamR1]" rMax="18.0*cm"/>
-   <ZSection z="0.02*cm" rMin="[cms:TrackBeamR1]" rMax="19.0*cm"/>
-   <ZSection z="89.9*cm" rMin="[cms:TrackBeamR1]" rMax="19.0*cm"/>
-   <ZSection z="89.9*cm" rMin="[cms:TrackBeamR1]" rMax="21.6*cm"/>
-   <ZSection z="99.00*cm" rMin="[cms:TrackBeamR1]" rMax="21.6*cm"/>
-   <ZSection z="99.00*cm" rMin="[cms:TrackBeamR1]" rMax="116.0*cm"/>
-   <ZSection z="[RootMidZ1]"   rMin="[cms:TrackLumiR1Min]" rMax="116.0*cm"/>
-   <ZSection z="[RootMidZ1]"   rMin="[cms:TrackLumiR1]" rMax="116.0*cm"/>
-   <ZSection z="[RootMidZ2]"   rMin="[cms:TrackLumiR1]" rMax="116.0*cm"/>
-   <ZSection z="[RootMidZ2]"   rMin="[cms:TrackLumiR2Min]" rMax="116.0*cm"/>
-   <ZSection z="[RootEndZ]"    rMin="[cms:TrackBeamR2]" rMax="116.0*cm"/>
-   <ZSection z="[RootEndZ]+[ZextensionCables]"    rMin="[cms:TrackBeamR2]" rMax="116.0*cm"/>
-   </Polycone>
-   <Polycone name="Phase2PixelEndcap" startPhi="0*deg" deltaPhi="360*deg" >
-   <ZSection z="[RootStartZ]"  rMin="[cms:TrackBeamR1]" rMax="[RootRadius]"/>
-   <ZSection z="[RootMidZ1]"   rMin="[cms:TrackBeamR1]" rMax="[RootRadius]"/>
-   <ZSection z="[RootMidZ1]"   rMin="[cms:TrackLumiR1]" rMax="[RootRadius]"/>
-   <ZSection z="[RootMidZ2]"   rMin="[cms:TrackLumiR1]" rMax="[RootRadius]"/>
-   <ZSection z="[RootMidZ2]"   rMin="[cms:TrackBeamR1]" rMax="[RootRadius]"/>
-   <ZSection z="[RootMidZ3]"   rMin="[cms:TrackBeamR1]" rMax="[RootRadius]"/>
-   <ZSection z="[RootEndZ]"    rMin="[cms:TrackBeamR2]" rMax="[RootRadius]"/>
-   </Polycone>-->
  <Polycone name="Forward" startPhi="0*deg" deltaPhi="360*deg" >    <!-- TEMPORARY!! Should be defined automatically -->
    <ZSection z="[RootStartZ]" rMin="2.8*cm" rMax="[RootRadius1]"/>
-   <ZSection z="99.00*cm" rMin="2.8*cm" rMax="[RootRadius1]"/>
-   <ZSection z="99.00*cm" rMin="2.8*cm" rMax="116.0*cm"/>
+   <ZSection z="95.90*cm" rMin="2.8*cm" rMax="[RootRadius1]"/>
+   <ZSection z="95.90*cm" rMin="2.8*cm" rMax="116.0*cm"/>
```

Comment:
@boudoul - Have you checked that this move in Z by 3.1 cm does not introduce an overlap? Or shall I do it? Thanks.